aiida/__init__.py

```diff
@@ -34,10 +34,8 @@
 __version__ = '1.5.0'
 __authors__ = 'The AiiDA team.'
 __paper__ = (
-    'G. Pizzi, A. Cepellotti, R. Sabatini, N. Marzari, and B. Kozinsky,'
-    '"AiiDA: automated interactive infrastructure and database for computational science", '
-    'Comp. Mat. Sci 111, 218-230 (2016); https://doi.org/10.1016/j.commatsci.2015.09.013 '
-    '- http://www.aiida.net.'
+    'S. P. Huber et al. AiiDA 1.0, a scalable computational infrastructure for automated reproducible workflows and '
+    'data provenance, Scientific Data 7, 300 (2020); https://doi.org/10.1038/s41597-020-00638-4'
 )
 __paper_short__ = 'G. Pizzi et al., Comp. Mat. Sci 111, 218 (2016).'
```

Comment:
Really sorry, I realised only now that there is a `__paper__` and a `__paper_short__`... 

So  one should update also the short one, and use the short or long one where appropriate. Also, note the comma after 'et al.' and I added the double quotes around the title, to keep the same styling as before and also because we have a lot of punctuation in our new title ;-)
I'm suggesting to keep anyway the `et al.` also for the long version that I think it's a good compromise.

```suggestion
    'S. P. Huber et al., "AiiDA 1.0, a scalable computational infrastructure for automated reproducible workflows and '
    'data provenance", Scientific Data 7, 300 (2020); https://doi.org/10.1038/s41597-020-00638-4'
)
__paper_short__ = 'S. P. Huber et al., Sci. Data 7, 300 (2020).'
```

Reply:
Haha no need to be sorry, cannot believe I completely overlooked that